cmd-dump.c

```diff
@@ -1238,8 +1238,10 @@ int command_dump(int argc, char *argv[], struct opts *opts)
 	struct ftrace_file_handle handle;
 
 	ret = open_data_file(opts, &handle);
-	if (ret < 0)
-		pr_err("cannot open data: %s", opts->dirname);
+	if (ret < 0) {
+		pr_warn("cannot open data: %s", opts->dirname);
+		return -1;
```

Comment:
Ditto.  Please add "%m" (and "\n") at the end so that users can identify what the problem is.